internal/pkg/filesystem/filesystem.go

```diff
@@ -209,23 +252,23 @@ func (f *Fs) ReadFile(path, desc string) (*model.File, error) {
 // WriteFile from string.
 func (f *Fs) WriteFile(file *model.File) error {
 	// Create dir
-	dir := filepath.Dir(string(file.Path))
```

Comment:
`FilePath` bol alias pre `string`, neodvedcilo sa to takto mat, takze je to obycajny `string`.